src/main/java/de/tum/in/www1/artemis/service/iris/model/IrisGPT3_5Service.java

```diff
@@ -118,6 +118,16 @@ public CompletableFuture<IrisMessage> getResponse(IrisSession session) {
         }
     }
 
+    /**
+     * Returns a format string used as the initial system prompt for storing in the database
+     *
+     * @return Initial system message for this LLM as format string
+     */
+    @Override
+    public String getInitialSystemMessageTemplate() {
+        return "This will be the initial system prompt...";
```

Comment:
I don't think making the intial system message model specific like this makes that much sense. This would mean a couple of things:
- Testing a new LLM is a lot of overhead, as you have to reengineer all prompts for this new LLM (or copy over the old ones)
- New Iris sub-features would not work with this system (e.g. see #6638) 
- Once we switch to Microsoft guidance this service will vanish and you would then have to put this message elsewhere 

Overall I think it makes sense to put it somewhere else for now (e.g. just IrisSessionService). In the future we probably want a full service just for managing these prompts, so that instructors can also modify them for a course/exercise if they want and maybe have optional LLM specific adaptations.